frontend/src/v5/store/tickets/tickets.types.ts

```diff
@@ -25,7 +25,9 @@ export interface PropertyDefinition {
 	type: PropertyTypeDefinition;
 	values?: string[] | 'jobsAndUsers' | 'riskCategories';
 	default?: string;
+	unique?: boolean;
 	readOnly?: boolean;
+	readOnlyOnUI?: boolean;
```

Comment:
Imagine a ticket with 2 properties: A (which is unique) and B.
If I try to update A to a value that already exists elsewhere, I get an error message. But if then I update B, should A be reverted to its previous value, so that B is updated? (At least, this what normally happens with error, such as with a required field).

Also, note that if I update A to a non-unique value and I try to update B, I get 2 error modals


Uploading 2024-03-13 17-58-34.mp4…